certified-connectors/smapOne/README.md

```diff
@@ -0,0 +1,31 @@
+# Publisher: smapOne AG
+# smapOne API
+The digital toolbox for successful companies
+With smapOne, companies can digitalise their processes. Team members can build, learn and try things out. After all, it Is they who are the true experts and can turn their own ideas into smaps.
+
+No programming knowledge needed!
+
+The smapOne API provides programmatic access to recorded data and datasources, making it possible to integrate with your business applications.
+
+## Prerequisites
+* A smapOne subscription is required, you can sign up on our Website:  [smapOne](https://platform.smapone.com/portal/Account/Register?AsCreator=True&lang=en-GB).
+
+## Obtaining Credentials
+
+* After signing up you can retrieve your API key directly from your profile
+[Profile](https://platform.smapone.com/portal/Creator/Profile/ManageApiToken)
+
+## Supported Operations
+
+The connector can use the most operations possible using smapOne v1 API and some from internal. 
+You can find more details about all operations and explanations about the various parameters here: [smapOne API Docs](https://platform.smapone.com/Backend/swagger/ui/index)
+
+### Get Data
+The main function allows exporting the recorded data. 
+Data can be exported as raw data in text format and as pdf or word documents
+
+### Manage Datasources
+With Datasources you can import master data to smapOne to use it in your smaps.
+

```

Comment:
Hello @strebbin 

Please add below sections.  Please refer the link below for your information. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

1. Known issues

Reply:
Hello @vmanoharas,

the Known Issues Section is added now.